modules/ROOT/attachments/cloud-api.yaml

```diff
@@ -4659,7 +4659,7 @@ info:
-  title: Redpanda Cloud API
+  title: Redpanda Cloud Control Plane API
```

Comment:
Let's wait till https://github.com/redpanda-data/docs/pull/929 is merged before we change the title.